PrebidMobile/PrebidMobile-core/src/main/java/org/prebid/mobile/rendering/sdk/InitializationNotifier.java

```diff
@@ -0,0 +1,94 @@
+package org.prebid.mobile.rendering.sdk;
```

Comment:
Not quite sure the relevance of it, but prebid classes have a copy of the license

```
/*
 *    Copyright 2018-2023 Prebid.org, Inc.
 *
 *    Licensed under the Apache License, Version 2.0 (the "License");
 *    you may not use this file except in compliance with the License.
 *    You may obtain a copy of the License at
 *
 *        http://www.apache.org/licenses/LICENSE-2.0
 *
 *    Unless required by applicable law or agreed to in writing, software
 *    distributed under the License is distributed on an "AS IS" BASIS,
 *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 *    See the License for the specific language governing permissions and
 *    limitations under the License.
 */
```

Reply:
We need to review this license for all classes and make automatic placement in an IDE. So for now it's not a problem.